PositionalGuide/ConfigWindow.cs

```diff
@@ -181,6 +197,24 @@ public class ConfigWindow: Window, IDisposable {
 
 		ImGui.PopStyleVar();
 
+		// Force circle colours either for all or defined individually for each
+		changed |= ImGui.Checkbox("Always use the defined colours for the circles?", ref circleColours[0]);
+		utils.Tooltip("This setting will cause the circle colours to be always the defined ones and disable the automatic colouring of the circle based on the guidelines.");
+
+		ImGui.PushStyleVar(ImGuiStyleVar.Alpha, circleColours[0] ? InactiveOptionAlpha : 1);
+
+		ImGui.Indent();
+		changed |= ImGui.Checkbox("Always use the defined color for the target circle?", ref circleColours[1]);
+		utils.Tooltip("As above but only for the target circle.");
```

Comment:
Please swap the tooltip call and the unindent call to match the rest of the rendering.